pydatastructs/linear_data_structures/linked_lists.py

```diff
@@ -61,7 +61,21 @@ def insert_after(self, prev_node, key, data=None):
         data
             Any valid data to be stored in the node.
         """
-        raise NotImplementedError('This is an abstract method')
```

Comment:
This is an abstract for the generic class `LinkedList`. Specific implementations of `insert_after` are there in subclasses. The reasons of having such abstract methods in parent classes can be found [here](https://stackoverflow.com/questions/8394252/what-is-the-point-of-using-abstract-methods). 
P.S. This is not a bug.